packages/web-components/src/components/va-alert/va-alert.scss

```diff
@@ -109,29 +109,29 @@ i.fa-times-circle::before {
 
 // Override USWDS background and border colors for VA styles.
```

Comment:
Can you update this comment or remove it? It doesn't make sense anymore if we're just using USWDS now.